bin/problem.py

```diff
@@ -675,10 +698,12 @@ def _validators(
         paths = [p for source_dir in cls.source_dirs for p in glob(problem.path / source_dir, "*")]
 
         # Handle default output validation
-        if cls == validate.OutputValidator and problem.settings.validation == "default":
-            if paths:
+        if cls == validate.OutputValidator:
+            if problem.settings.is_legacy() and not problem.custom_output and paths:
                 error("Validation is default but custom output validator exists (ignoring it)")
-            paths = [config.tools_root / "support" / "default_output_validator.cpp"]
+                paths = []
+            if not paths:
```

Comment:
it's checked elsewhere that `custom_output` requires at least one to be present and non-empty `path` here, right?

Reply:
Yep, that should happen in `_read_settings`. Reminder to self to add some `assert`ion for that here 🙂 

(Also note that the new spec does not have an explicit type for custom output, this is determined by the existance of the output validator)